podcasts/TranscriptsViewController.swift

```diff
@@ -361,6 +402,77 @@ class TranscriptsViewController: PlayerItemViewController {
         }
     }
 
+    // MARK: - Search
+
+    func performSearch(_ term: String) {
+        Task {
+            findOccurrences(of: term)
+            updateNumberOfResults()
+            refreshText()
+            scrollToFirstResult()
+        }
+    }
+
+    func findOccurrences(of term: String) {
+        guard let transcriptText = transcript?.attributedText.string,
+              !term.isEmpty else {
+            resetSearch()
+            return
+        }
+
+        let kmpSearch = KMPSearch(pattern: term)
+        searchIndicesResult = kmpSearch.search(in: transcriptText)
+        searchTerm = term
+    }
+
+    @MainActor
+    func updateNumberOfResults() {
+        guard searchIndicesResult.count > 0 else {
```

Comment:
Same here about the `guard`

Reply:
Done in b351ceeb690ee61d6f813f00e0b91012daa36bff